config/default.yml

```diff
@@ -427,6 +427,15 @@ PreCommit:
     required_executable: 'grep'
     flags: ['-IHn', "^<<<<<<<[ \t]"]
 
+  MetadataJsonLint:
+    enabled: false
+    strict_license: true
+    strict_dependencies: false
+    fail_on_warning: true
+    description: 'Checking Puppet module metadata'
+    required_executable: 'metadata-json-lint'
+    install_command: 'gem install metadata-json-lint semantic_puppet'
```

Comment:
Would suggest we remove `semantic_puppet` from this list, as the `post_install` message for `metadata-json-lint` explicitly mentions installing it if you're on an older version of Puppet: https://github.com/voxpupuli/metadata-json-lint/blob/a57b7d2f21d746d616ef6032624df1a7cc183425/metadata-json-lint.gemspec#L26-L30